linera-views-derive/src/lib.rs

```diff
@@ -311,7 +311,10 @@ fn generate_graphql_code_for_field(field: Field) -> (TokenStream2, Option<TokenS
             };
             (r#impl, None)
         }
-        "CollectionView" | "CustomCollectionView" => {
+        "CollectionView"
```

Comment:
I look forward to the solution of https://github.com/linera-io/linera-protocol/issues/678